packages/graphql-hooks/index.d.ts

```diff
@@ -183,10 +187,20 @@ export interface UseClientRequestOptions<ResponseData = any, Variables = object>
   client?: GraphQLClient
 }
 
+export type RefetchAferMutationsData = {
+  mutation: string
+  filter?: (variables: object) => boolean
+}
+
 export interface UseQueryOptions<ResponseData = any, Variables = object>
   extends UseClientRequestOptions<ResponseData, Variables> {
   ssr?: boolean
   skip?: boolean
+  refetchAfterMutations?:
+    | string
+    | string[]
+    | RefetchAferMutationsData
+    | RefetchAferMutationsData[]
 }
```

Comment:
why don't we define the RefetchAfterMutationsData as a union type which represents all the possible options instead of doing it here?